service/controller/resource/values/create.go

```diff
@@ -21,6 +21,15 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	configVersion, ok := app.GetAnnotations()[annotation.ConfigVersion]
 	if !ok {
 		r.logger.Debugf(ctx, "App CR %q is missing %q annotation", app.Name, annotation.ConfigVersion)
+		if _, ok := app.GetAnnotations()[PauseAnnotation]; ok {
+			r.logger.Debugf(ctx, "clearing %q annotation from App CR %#q", PauseAnnotation, app.Name)
+			app.SetAnnotations(removeAnnotation(app.GetAnnotations(), PauseAnnotation))
+			err = r.k8sClient.CtrlClient().Update(ctx, &app)
+			if err != nil {
+				return microerror.Mask(err)
+			}
+			r.logger.Debugf(ctx, "cleared %q annotation from App CR %#q", PauseAnnotation, app.Name)
+		}
```

Comment:
I think we should do the same during deletion. Otherwise the CR has a chance to never be cleared.